### PR TITLE
Input src change with Capslock, Modifier+backspace to delete, Leopold style Page control

### PR DESCRIPTION
**Pull request by Namu-Lee**
1. IME: Caps Lock to ctrl+shift+space
2. Modifier(shift/option)+Backsapce to Delete
3. right option+arrow to pageup/pagedown/home/end

### CODE REVIEW

**Comment by MuhammedZakir** [comment]
Can you please format the json files and also update the branch?

**Comment by Namu-Lee** [comment]
done!

File: public/json/backspace_to_delete.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+						"right_alt"
+					 ],
+					 "optional":[
+						"caps_lock"
+					 ]
+				  }
+			   },
+			   "to":[
+				  {
+					 "repeat":true,
```
Comment (MuhammedZakir) [inline public/json/backspace_to_delete.json:22]: Please remove this. `repeat` is `true` by default.

File: public/json/backspace_to_delete.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+					 ],
+					 "optional":[
+						"caps_lock"
+					 ]
+				  },
+				  "key_code":"delete_or_backspace"
+			   },
+			   "to":[
+				  {
+					 "repeat":true,
```
Comment (MuhammedZakir) [inline public/json/backspace_to_delete.json:50]: This too.

**Review by MuhammedZakir — CHANGES_REQUESTED** [review]

**Comment by MuhammedZakir** [comment]
Other than those 2, everything else is okay. Thank you!

**Comment by Namu-Lee** [comment]
Removed all "repeat: true". thanks.

File: public/json/option+arrow_to_pagecontrol.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+			  "optional": [
+				"caps_lock"
+			  ]
+			},
+			"key_code": "up_arrow"
+		  },
+		  "to": [
+			{
+			  "key_code": "page_up",
+			  "modifiers": []
```
Comment (MuhammedZakir) [inline public/json/option+arrow_to_pagecontrol.json:23]:
```suggestion
			  "key_code": "page_up"
```

File: public/json/option+arrow_to_pagecontrol.json [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+			  "optional": [
+				"caps_lock"
+			  ]
+			},
+			"key_code": "down_arrow"
+		  },
+		  "to": [
+			{
+			  "key_code": "page_down",
+			  "modifiers": []
```
Comment (MuhammedZakir) [inline public/json/option+arrow_to_pagecontrol.json:48]:
```suggestion
			  "key_code": "page_down"
```

File: public/json/option+arrow_to_pagecontrol.json [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+			  "optional": [
+				"caps_lock"
+			  ]
+			},
+			"key_code": "left_arrow"
+		  },
+		  "to": [
+			{
+			  "key_code": "home",
+			  "modifiers": []
```
Comment (MuhammedZakir) [inline public/json/option+arrow_to_pagecontrol.json:73]:
```suggestion
			  "key_code": "home"
```

File: public/json/option+arrow_to_pagecontrol.json [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+			  "optional": [
+				"caps_lock"
+			  ]
+			},
+			"key_code": "right_arrow"
+		  },
+		  "to": [
+			{
+			  "key_code": "end",
+			  "modifiers": []
```
Comment (MuhammedZakir) [inline public/json/option+arrow_to_pagecontrol.json:98]:
```suggestion
			  "key_code": "end"
```

**Comment by MuhammedZakir** [comment]
There was also a blank `to.modifiers`. That is also not needed. I have removed that myself.

**Review by MuhammedZakir — APPROVED** [review]

**Comment by MuhammedZakir** [comment]
Merged. :-)

**Pull request merged**